src/main/java/org/broadinstitute/hellbender/tools/exome/TargetArgumentCollection.java

```diff
@@ -31,7 +32,7 @@
  * </p>
  * <p>
  *   If even this mechanism does not provide a target file name, and the target file is deemed non-optional,
- *   the a {@link UserException.BadArgumentValue} is thrown.
+ *   the a {@link CommandLineException.BadArgumentValue} is thrown.
```

Comment:
Sounds like we might be able to work around the issue by tweaking the way javadoc is run, then, instead of removing all links to external classes from the docs (if we care about the javadoc output at some point).